CHANGES

```diff
@@ -23,6 +23,7 @@ Features added
 * Show warnings if no domains match with `primary_domain` (ref: #2001)
 * C++, show warnings when the kind of role is misleading for the kind
   of target it refers to (e.g., using the `class` role for a function).
+* public names for latex environments and parameters used by note, warning, and other admonition types, allowing full customizability from either ``'preamble'`` key or ``raw`` directive (ref: feature request #2674)
```

Comment:
ok thanks. I am all about folding (I noticed that Travis-CI has a cap at 95 (or 96?) for python files, I am not sure about CHANGES). As I am used to folding at 72 or 80, I thought from long lines in CHANGES Bug fixes entries that no folding was required.